Source/WebCore/accessibility/AccessibilityNodeObject.h

```diff
@@ -198,8 +198,8 @@ class AccessibilityNodeObject : public AccessibilityObject {
     HTMLLabelElement* labelElementContainer() const;
 
     String ariaAccessibilityDescription() const;
-    Vector<Element*> ariaLabeledByElements() const;
-    String descriptionForElements(Vector<Element*>&&) const;
+    Vector<Ref<Element>> ariaLabeledByElements() const;
+    String descriptionForElements(Vector<Ref<Element>>&&) const;
```

Comment:
I wonder why this needs to adopt a vector; it’s serializing the elements, not storing references to them. I’d think this would be `const Vector<Ref<Element>>&` or `Span<const Ref<Element>>`.